### PR TITLE
fix(state): match FF's getState output

### PR DESCRIPTION
**Pull request by rvsia**
part of https://github.com/data-driven-forms/react-forms/issues/705

### CODE REVIEW

File: packages/form-state-manager/src/types/manager-api.d.ts [hunk 1]
```diff
@@ -50,8 +50,8 @@ export interface ManagerState {
   fieldListeners: AnyObject;
   active: string | undefined;
   dirty: boolean;
-  dirtyFields: Array<string>;
-  dirtyFieldsSinceLastSubmit: Array<string>;
+  dirtyFields: AnyObject;
+  dirtyFieldsSinceLastSubmit: AnyObject;
```
Comment (Hyperkid123) [inline packages/form-state-manager/src/types/manager-api.d.ts:54]: This should probably be a boolean object
```ts
interface Foo {
  [key: string]: boolean
}
```

File: packages/form-state-manager/src/utils/manager-api.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import CreateManagerApi, { ManagerState, ManagerApi, AsyncWatcher, AsyncWatcherR
 import AnyObject from '../types/any-object';
 import FieldConfig from '../types/field-config';
 import { formLevelValidator } from './validate';
+import { values } from 'lodash';
```
Comment (Hyperkid123) [inline packages/form-state-manager/src/utils/manager-api.ts:8]: Maybe I am blind, but I can't see this used anywhere. If it is, please use direct import `lodash/values`. Or consider using `Object.values` instead.

File: packages/form-state-manager/src/types/any-object.d.ts [hunk 3]
```diff
@@ -1,3 +1,7 @@
+export interface AnyBooleanObject {
+  [key: string]: any;
```
Comment (Hyperkid123) [inline packages/form-state-manager/src/types/any-object.d.ts:2]: should be `:boolean` :wink:

Comment (rvsia) [reply packages/form-state-manager/src/types/any-object.d.ts]: ..
...
....
Yep... it was made to test you. :smile:

**Pull request merged**